relay/network_graph/graph.py

```diff
@@ -718,12 +750,14 @@ def close_trustline_path_triangulation(
             )  # don't check max_hops, we know we're below
             cost = cost_accumulator.compute_cost_for_path(self.graph, path)
         except nx.NetworkXNoPath:
-            return PaymentPath(fee=0, path=[], value=value)
+            return PaymentPath(fee=0, path=[], value=value, fee_payer=FeePayer.SENDER)
 
         if balance < 0:
             path.reverse()
 
-        return PaymentPath(fee=cost[0], path=path, value=value)
+        return PaymentPath(
+            fee=cost[0], path=path, value=value, fee_payer=FeePayer.SENDER
```

Comment:
This should not always return sender. It should return receiver, if the receiver has to pay for this transfer. You might argue that this is not necessary because sender and receiver are the same person, but if you want to use this to settle and you will just use a normal transfer, you need to know if you use transfer where the sender pays or where the receiver pays, so we should just return here the right value.

Reply:
I changed it.